nx/include/switch/services/nifm.h

```diff
@@ -34,6 +47,12 @@ Result nifmIsWirelessCommunicationEnabled(bool* out);
  */
 Result nifmSetWirelessCommunicationEnabled(bool enable);
 
+/**
+ * @note Will fail with 0xd46ed if Internet is neither connecting or connected (airplane mode or no known network in reach).
+ * @param wifiStrength Strength of the Wi-Fi signal in number of bars from 0 to 3.
+ */
+Result nifmGetInternetConnectionStatus(NifmInternetConnectionType* connectionType, u8* wifiStrength, NifmInternetConnectionStatus* connectionStatus);
```

Comment:
u32* for wifiStrength.

Reply:
SwIPC says that all three parameters are u8, do you still want me to change it to u32?

Reply:
Just the output param, let the IPC one alone.